Fatal error during cdk deploy: selectedItems is declared but its value is never read.

adding: genai_core/clients.py (deflated 65%)
Bundling asset GenAIChatBotStack/UserInterface/UserInterfaceDeployment/Asset1/Stage...
src/components/chatbot/sessions.tsx:174:32 - error TS6133: 'selectedItems' is declared but its value is never read.

174           itemSelectionLabel: ({ selectedItems }, item) => item.title!, // eslint-disable-line @typescript-eslint/no-unused-vars
                                   ~~~~~~~~~~~~~~~~~


Found 1 error in src/components/chatbot/sessions.tsx:174

Error: Command failed: npm --silent --prefix "/home/ubuntu/environment/aws-genai-llm-chatbot/lib/user-interface/react-app" run build
    at checkExecSyncError (node:child_process:890:11)
    at execSync (node:child_process:962:15)
    at Object.tryBundle (/home/ubuntu/environment/aws-genai-llm-chatbot/lib/user-interface/index.ts:184:23)
    at AssetStaging.bundle (/home/ubuntu/environment/aws-genai-llm-chatbot/node_modules/aws-cdk-lib/core/lib/asset-staging.js:2:33)
    at AssetStaging.stageByBundling (/home/ubuntu/environment/aws-genai-llm-chatbot/node_modules/aws-cdk-lib/core/lib/asset-staging.js:1:5297)
    at stageThisAsset (/home/ubuntu/environment/aws-genai-llm-chatbot/node_modules/aws-cdk-lib/core/lib/asset-staging.js:1:2728)
    at Cache.obtain (/home/ubuntu/environment/aws-genai-llm-chatbot/node_modules/aws-cdk-lib/core/lib/private/cache.js:1:242)
    at new AssetStaging (/home/ubuntu/environment/aws-genai-llm-chatbot/node_modules/aws-cdk-lib/core/lib/asset-staging.js:1:3125)
    at new Asset (/home/ubuntu/environment/aws-genai-llm-chatbot/node_modules/aws-cdk-lib/aws-s3-assets/lib/asset.js:1:1080) {
  status: 2,
  signal: null,
  output: [ null, null, null ],
  pid: 5592,
  stdout: null,
  stderr: null
}

--- a/lib/user-interface/react-app/src/components/chatbot/sessions.tsx
+++ b/lib/user-interface/react-app/src/components/chatbot/sessions.tsx
@@ -20,7 +20,7 @@ import RouterButton from "../wrappers/router-button";
 import { Session } from "../../API";
 
 export interface SessionsProps {
-  toolsOpen: boolean;
+  readonly toolsOpen: boolean;
 }
 
 export default function Sessions(props: SessionsProps) {
@@ -171,7 +171,8 @@ export default function Sessions(props: SessionsProps) {
             `${selectedItems.length} ${
               selectedItems.length === 1 ? "item" : "items"
             } selected`,
-          itemSelectionLabel: ({ selectedItems }, item) => item.title!, // eslint-disable-line @typescript-eslint/no-unused-vars
+          // @ts-expect-error no-unused-var
+          itemSelectionLabel: (e, item) => item.title!,
         }}
         pagination={<Pagination {...paginationProps} />}
         loadingText="Loading history"

